README.md

```diff
@@ -6,11 +6,39 @@ Assumes that the database itself has already been created and configured.  At a
 * `imrt_ingest`
 * `imrt_search`
 
+## Create `imrt_ingest` and `imrt_search` Users
```

Comment:
This SQL could also likely be added to a `beforeMigrate.sql` to be executed by flyway.  We'd have to wrap the `CREATE USER` steps in an `IF` statement to check for existence first, but that could be done without too much effort.  The reason I didn't do it already is I think it's more evident/obvious to a new developer to have this SQL in the README.

But if we want, it wouldn't be too much effort to add this to a `beforeMigrate.sql` and instruct the new developer to open that file & choose a password.

Reply:
I think the way you have it is good.  Now that these are here can you make sure the other IMRT projects reference this project's readme rather than provide the steps?   For example, it seems like your steps replace the need for these steps in the ingest service and instead the ingest service can link to this readme? https://github.com/SmarterApp/AP_IMRT_ItemIngestService#create-database-and-users

Reply:
Definitely.  There are some steps in that README that aren't here.  I can merge all the steps together; I like the idea of having all the database creation steps in one place.  I'll make those updates.